.github/workflows/notebook-test-suite.yml

```diff
@@ -0,0 +1,176 @@
+# This workflow runs Jupyter notebook based regression test suites.
+# It can be run manually run on github Actions or invoked through
+# the github Action workflow api.
+# When invoked through the github Action workflow api, a event type needs
+# to be provided. The event type should be either the name of one of the
+# Harmony services or 'all'.
+# The notebook test suites associated with Harmony service matching the event
+# will be executed, or all the notebook test suites will be exectued when 'all'
+# is provided which is triggered when a Harmonny server deployment occurs.
+#
+# Required settings in the repository:
+# Environments (exactly the following strings):
+# - prod
+# - uat
+#
+# Secrets in each environment:
+# - EDL_USER
+# - EDL_PASSWORD
+#
+# Note: A workflow_call event can only have inputs of type boolean, number or
+# string.
+name: Run Jupyter notebook based test suite
+
```

Comment:
Was the work to make a more dynamic [run-name](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#run-name) going to be included in another PR?

Reply:
This is what the workflow result page looks like:
![Screenshot 2024-10-04 at 12 47 36 PM](https://github.com/user-attachments/assets/f4dc84c1-14ac-4781-9550-774ea5c560dd)
I think it is pretty clear without the `run-name`. If you want the step to be something else using run-name, I would like that to be in a separate PR.

Reply:
oh, I misunderstood run-name. I have added it.